Library/Formula/mono.rb

```diff
@@ -41,10 +51,11 @@ def install
   end
 
   test do
+    ohai "Remember to set MONO_GAC_PREFIX=\"#{HOMEBREW_PREFIX}\" to finalize the installation."
```

Comment:
No, I thought that if the user goes through the trouble of doing a 'test mono' he/she really wants to debug a problem, and this is the most likely problem for this formula.